lib/IRGen/GenProto.cpp

```diff
@@ -1792,10 +1789,8 @@ class AccessorConformanceInfo : public ConformanceInfo {
       if (IGM.Context.LangOpts.hasFeature(Feature::Embedded)) {
         // In Embedded Swift associated-conformance entries simply point to the witness table
         // of the associated conformance.
-        ProtocolConformanceRef assocConf =
-          SILWT->getConformance()->getAssociatedConformance(requirement.getAssociation(),
-                                                            requirement.getAssociatedRequirement());
-        llvm::Constant *witnessEntry = IGM.getAddrOfWitnessTable(assocConf.getConcrete());
+        ProtocolConformance *assocConf = associatedWitness.Witness.getConcrete();
```

Comment:
`associatedWithness` comes from a variable which is part of that `#ifndef NDEBUG` above. Makes this a compilation failure when compiling a non-asserts toolchain.

Is there a fix already happening for this?